docs/building-chain/depinject.md

```diff
@@ -0,0 +1,17 @@
+<!--
+order: 1
+-->
+
+# Dependency Injection
```

Comment:
This will be added after https://github.com/cosmos/cosmos-sdk/issues/12633. I am for keeping this boilerplate, as it permits me to link the testing documentation to here.